client/verta/verta/client.py

```diff
@@ -644,160 +643,160 @@ def get_or_create_repository(self, name=None, workspace=None, id=None, public_wi
         else:
             raise ValueError("must specify either `name` or `id`")
 
-    # NOTE: dataset visibility cannot be set via a client
-    def set_dataset(self, name=None, type="local",
```

Comment:
I'm intending to later uncomment & rename these old `client` dataset methods, as a temporary option for users who are using the old API but don't have immediate time to migrate their workflows over

Reply:
That means I'll also need to bring back the deleted `_dataset.py`...but it's Just Temporary™

Reply:
Holy Molly that's a lot of commenting! I might be missing it, but I think you'll also have to comment out or remove [the tests for this old dataset API](https://github.com/VertaAI/modeldb/blob/master/client/verta/tests/test_datasets.py) right?

Reply:
Actually maybe it still works, if the new implementation keeps the same client interface? Idk 🤔

Reply:
The old tests will need to be adjusted/removed, but that will come later.

Reply:
What would it take to support the old API in the new style? Not re-using a lot of the old logic (i.e. pretend those fields have been removed). Maybe it's not worth doing, but just considering the options.

Reply:
My plan is to explore this today—the main puzzle piece I'm considering is for `Dataset.create_version()` to check if the first arg is a `str`/`list<str>` (which means it's being used old-style, instead of a blob), and make some `isfile()`/`boto3` checks to generate the content blob internally.